packages/dashboard/src/store/state.ts

```diff
@@ -1,6 +1,8 @@
 import { DashboardConfiguration, Widget } from '../types';
+import { ListAssetsResponse } from '@aws-sdk/client-iotsitewise';
 
 export type DashboardState = {
+  query: null | ListAssetsResponse;
```

Comment:
Is this the query result from the executed query passed into the internal dashboard?

Reply:
Ah, I think I forgot to remove this. I don't think we should put the query in dashboard state at all because the source-iotsitewise provider tracks its state internally. I will fix.